### ISSUE TITLE
Ensure support ability to pre-create hostPath persistent volumes.

### ISSUE THREAD

**Issue by GrahamDumpleton** [openshift-evangelists/openshifter#8]
Old version supports:

```
disks:
  pvs: 100                    # optional, number of GB for PersistentVolumes

components:                   # optional, components to setup on the cluster
  pvs: false                  # optional, PersistentVolumes
  hostPath: false             # optional, allow HostPath

pvs:                          # required if PersistentVolumes components is enabled
  type: host                  # optional, PersistentVolume technology to use
  count: 0                    # optional, how many PersistentVolumes to create
  size: 5                     # required, PersistentVolume size in GB
```

**Comment by GrahamDumpleton** [comment]
And try and ensure you do this in a optimal way, unlike old version. Try and not to use a separate remote exec to create the directory for each separate volume, nor create persistent volume resource one at a time. When you have large number of volumes that can take a long time. Better to see if can invoke one remote command which creates all directories from host in one pass etc.

**Comment by marekjelen** [comment]
Fixed in 05765e9a1889f701384daa5ebffa2f1c80ba05a0

```
disks:
  pvs: 100                    # optional, number of GB for PersistentVolumes
```

is deprecated

```
components: 
  pvs: true
```

enables PVs (right now always hostPath)

```
components:
  hostPath: false             # optional, allow HostPath
```

is deprecated (it was actually not related to PVs)

```
pvs:
  type: gluster (optional)
  count: 1
  size: 5 
```

creates one PV with the size of 5GB. If `type: gluster` is omitted, it will create simple hostPath PV in /pvs. If `type: gluster` is specified, it will deploy "pvs instance" with Gluster (1 brick, 1 volume) on it and will mount it on master/infra/node instances as /pvs ... so when the PVs are being created, will be backed by Gluster (thus allowing RWX access mode across cluster) without the hassle to setup proper Gluster PVs.

The efficiency is tracked in separate issue.
